west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 795ac34f291c2f9895e7a3a03eafc053ad1d36f2
+      revision: pull/77289/head
```

Comment:
Still needed - or has west been updated now ?

Reply:
@lgirdwood https://github.com/zephyrproject-rtos/zephyr/pull/77289 still under review, waiting for the assignee in particular